bnlrubyshops.csv

```diff
@@ -24,17 +23,17 @@ Hstry,http://hstry.co/,product,Brussels,Belgium
 Intuo,http://intuo.io,product,Ghent,Belgium
 JDi Agency,https://jdi-agency.com/,agency,Brussels,Belgium
 ListMinut,https://listminut.be/workforus,product,Brussels,Belgium
+LichtVis,http://lichtvis.be,agency,Brugge,Belgium
 Le Wagon,http://www.lewagon.org/brussels,training,Brussels,Belgium
-MyMicroInvest,https://www.mymicroinvest.com/en,product,Brussels,Belgium
+Spreds,https://www.spreds.com/,product,Brussels,Belgium
 NephroFlow,https://nephroflow.com,product,Bruges,Belgium
 Neveo,http://www.neveo.be/,,Louvain-la-Neuve,Belgium
 Nimbly,http://nimbly.be,,Leuven,Belgium
 Ontoforce,http://www.ontoforce.com,,Ghent,Belgium
-Openminds,http://openminds.be,hosting,Ghent,Belgium
 Phonoid,http://phonoid.com,agency,Mons,Belgium
 PlayPass,http://www.playpass.be,product,Antwerp,Belgium
-PullReview,https://www.pullreview.com/,product,Brussels,Belgium
-RubyMotion,http://www.rubymotion.com/,product,Liège,Belgium
+Pootsy,https://pootsy.be/,product,Brussels,Belgium
+Prospect,https://prospect.io/,product,Brussels,Belgium
```

Comment:
tbc, but I think the brand is actually `Prospect.io`
![image](https://user-images.githubusercontent.com/1906401/46933888-21584d00-d056-11e8-9451-11deddc9fdd4.png)